index.md

```diff
@@ -58,104 +58,105 @@ This page introduces guidelines and other standards related to web accessibility
 
-- [Timed Text Markup Language (TTML)](https://www.w3.org/TR/ttml/) is intended to be used for transcoding or exchanging timed text information among legacy distribution content formats for subtitling and captioning.
+- [Timed Text Markup Language (TTML)](https://www.w3.org/TR/ttml/) sirve para transcodificar o intercambiar información textual sincronizada temporalmente entre formatos de subtítulos antiguos.
 
-### Evaluation {#eval}
+### Evaluación {#eval}
 
-The following resources support development of accessibility evaluation methods and tools:
+Los siguientes recursos permiten desarrollar métodos y herramientas para evaluar la accesibilidad:
 
-- [[Accessibility Conformance Testing (ACT) Overview]](/standards-guidelines/act/) &mdash; ACT establishes and documents rules for testing the conformance of web content to accessibility standards.
+- [[Resumen de Accessibility Conformance Testing (ACT)]](/standards-guidelines/act/) &mdash; ACT establece y describe normas para comprobar la conformidad del contenido web con los estándares de accesibilidad.
 
-- [[Evaluation and Report Language (EARL) Overview]](/standards-guidelines/earl/) &mdash; EARL is a machine-readable format for expressing test results.
+- [[Resumen de Evaluation and Report Language (EARL)]](/standards-guidelines/earl/) &mdash; EARL es un formato para ser leído por aplicaciones que sirve para expresar los resultados de las pruebas.
 
-Additional resources related to evaluation are listed in the [[Evaluating Web Accessibility Overview]](/test-evaluate/), including:
+Recursos adicionales relacionados con la evaluación se pueden encontrar en el [[resumen de evaluación de la accesibilidad web]](/test-evaluate/), incluyendo:
 
-- [[WCAG-EM Overview: Website Accessibility Conformance Evaluation Methodology]](/test-evaluate/conformance/wcag-em/) &mdash; WCAG-EM is an approach for determining how well a website conforms to Web Content Accessibility Guidelines (WCAG).
+- [[Resumen de WCAG-EM: Website Accessibility Conformance Evaluation Methodology]](/test-evaluate/conformance/wcag-em/) &mdash; WCAG-EM es una aproximación para determinar cuánto se ajusta un sitio web a las pautas de accesibilidad para el contenido web (WCAG).
 
-### Personalization
+### Personalización
 
-[[Personalization Overview]](/personalization/) &mdash; Personalization involves tailoring the user experience to meet the needs and preferences of the individual user. Content authors can use personalization standards to provide a default design and enable user personalization with minimal work.
+[[Resumen sobre la personalization]](/personalization/) &mdash; La personalización consiste en ajustar la experiencia del usuario a fin de cubrir las necesidades y preferencias individuales. Los autores de contenido pueden usar estándares de personalización para proporcionar un diseño por defecto y permitir la personalización del usuario con un esfuerzo mínimo.
```

Comment:
It seems that the sentence is written in Spanish but leaving the word "personalization" in English. I think it should be translated.